src/pybind/mgr/mgr_module.py

```diff
@@ -59,22 +59,41 @@ def emit(self, record):
         self._module._ceph_log(ceph_level, self.format(record))
 
 
-def configure_logger(module_inst, name):
-    logger = logging.getLogger(name)
+def configure_logger(module_inst, module_name):
+    """
+    Create and configure the logger with the specified module.
+
+    A handler will be added to the root logger which will redirect the
+    messages from all loggers (incl. 3rd party libraries) to the Ceph log.
 
-    # Don't filter any logs at the python level, leave it to C++
-    logger.setLevel(logging.DEBUG)
+    :param module_inst: The module instance.
+    :type module_inst: instance
+    :param module_name: The module name.
+    :type module_name: str
+    :return: Return the logger with the specified name.
+    """
+    logger = logging.getLogger(module_name)
+    # Don't filter any logs at the python level, leave it to C++.
+    logger.setLevel(logging.NOTSET)
 
     # FIXME: we should learn the log level from C++ land, and then
```

Comment:
afaik this FIXME belongs to `logger.setLevel` above

Reply:
Fixed